src/main/java/org/kohsuke/github/GHCommentChanges.java

```diff
@@ -0,0 +1,40 @@
+package org.kohsuke.github;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+/**
+ * Wrapper for changes on issue and pull request review comments action="edited"
+ *
+ * @see GHEventPayload.IssueComment
+ * @see GHEventPayload.PullRequestReviewComment
+ */
+@SuppressFBWarnings(value = "UWF_UNWRITTEN_FIELD", justification = "JSON API")
+public class GHCommentChanges {
```

Comment:
I wonder if this class should be a static nested class in `GHEventPayload`. This class is getting big but I think it makes more sense than having it as a top level class.

Let's wait for @bitwiseman 's opinion before changing anything though.

Reply:
@gsmet - I agree with you here actually, especially since `GHCommentChanges` is only applicable to event payloads. But I'll also wait for @bitwiseman's feedback before making any changes.

Perhaps others like `GHIssueChanges` also need to be refactored in this way? (in a separate PR)

Reply:
Agreed, this is only used by events, it should go in `GHEventPayload`.

Reply:
Also, as a static nested class it can be called `CommentChanges` without the `GH`.

Reply:
Made these changes, please see latest commit. Thanks!